nemo_skills/pipeline/generate.py

```diff
@@ -272,8 +306,13 @@ def generate(
     with run.Experiment(expname) as exp:
         extra_arguments_original = extra_arguments
         # TODO: reduce code duplication
+        donefiles = defaultdict(list)
         if random_seeds:
             for chunk_id in chunk_ids:
+                for seed in random_seeds:
```

Comment:
Do we need this logic here? Why not do everything inside get_cmd, so that you don't need to pass any extra arguments there?